cmd/dcrdata/internal/explorer/templates.go

```diff
@@ -369,6 +369,9 @@ func makeTemplateFuncMap(params *chaincfg.Params) template.FuncMap {
 		"intMultiply": func(a, b int) int {
 			return a * b
 		},
+		"currentYear": func() string {
+			return fmt.Sprint(time.Now().Year())
```

Comment:
`fmt.Sprint` should basically never be used for trivial numeric conversions because it is significantly more expensive (at least 2x in the best case) than `strconv.Itoa`, both in terms of CPU usage.  It also needlessly forces an additional allocation.

```Go
return strconv.Itoa(time.Now().Year())
```

This is fairly trivial to show via a benchmark with 10 iterations on both variants:


```
name   old time/op    new time/op    delta
---------------------------------------------------------------
Year   92.3ns ± 3%    40.4ns ± 2%    -56.23%  (p=0.000 n=10+10)

name   old alloc/op   new alloc/op   delta
---------------------------------------------------------------
Year   16.0B ± 0%     4.0B ± 0%      -75.00%  (p=0.000 n=10+10)
```

Reply:
Thanks, I've updated the PR.